queries.sql

```diff
@@ -8,3 +8,64 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg between 10.4 and 17.3;
+
+--  Transaction
+
+BEGIN;
+
+UPDATE animals SET species = 'unspecified' WHERE neutered = false;
+
+ROLLBACK;
```

Comment:
- [ ] in this transaction you don't need to specify if the animal was neutered or not
- [ ] please make sure the verify if the changes where made after the update and if the changes were undone after the rollback

Reply:
@mikethreels, I verified the changes before roll back but I don't know if there is another way you want me to do it, It'll be helpful if you can be more explicit with what you want me to do. Thanks